modules/supported-drift-metrics.adoc

```diff
@@ -0,0 +1,38 @@
+:_module-type: REFERENCE
+
+[id="supported-drift-metrics_{context}"]
+= Supported drift metrics
+
+{productname-long} supports the following data drift metrics.
+
+MeanShift::
+
+The MeanShift metric calculates the per-column probability that the data values in a test data set are from the same distribution as those in a training data set, assuming that the values are normally distributed. This metric works by measuring the difference in the means of specific features between the two datasets. 
++
+MeanShift is useful for identifying straightforward changes in data distributions, such as when the entire distribution has shifted to the left or right along the feature axis.
++
+This metric returns the probability that the distribution seen in the "real world" data comes from the same distribution as the reference data. The closer the value is to 0, the more likely there is to be significant drift.
+
+FourierMMD::
+
+The FourierMMD metric provides the probability that the data values in a test data set have drifted from the training data set distribution, assuming that the computed Maximum Mean Discrepancy (MMD) values are normally distributed. This metric works by comparing the empirical distributions of the data sets by using an MMD measure in the Fourier domain. 
++
+FourierMMD is useful for detecting subtle shifts in data distributions that might be overlooked by simpler statistical measures.
++
+This metric returns the probability that the distribution seen in the "real world" data has drifted from the reference data. The closer the value is to 1, the more likely there is to be significant drift.
+
+KSTest::
+
+The KSTest metric calculates two Kolmogorov-Smirnov tests for each column to determine whether the data sets are drawn from the same distributions. This metric works by measuring the maximum distance between the empirical cumulative distribution functions (CDFs) of the data sets, without assuming any specific underlying distribution.
```

Comment:
You could omit "works by" from: 

 This metric **works by** measuring the maximum distance between the empirical cumulative distribution functions (CDFs) of the data sets, without assuming any specific underlying distribution.
 
 Instead use "This metric measures..."